src/providers/BackgroundReplacementProvider/docs/useBackgroundReplacement.stories.mdx

```diff
@@ -0,0 +1,114 @@
+import { Props } from '@storybook/addon-docs/blocks';
+import { useBackgroundReplacement } from '../';
+
+<Meta title="SDK Hooks/useBackgroundReplacement" />
+
+# useBackgroundReplacement
+
+The `useBackgroundReplacement` hook returns a function transforming a normal video device to a `DefaultVideoTransformDevice`, and also a 
+state called `isBackgroundReplacementSupported` which indicates whether or not background replacement processor is fully loaded and ready to be used. 
+
+You should see either "processor is supported" or "processor is not supported" in your browser developer tools based on whether or not
+background replacement is supported on your device and browser version. For more information on if background replacement is supported, refer
+to [Amazon Chime SDK for JavaScript Background replacement Guide](https://github.com/aws/amazon-chime-sdk-js/blob/main/guides/15_Background_Filter_Video_Processor.md#integrating-background-filters-into-your-amazon-chime-sdk-for-javascript-application).
+
+You can check whether or not the processor has been loaded correctly by checking the state of `isBackgroundReplacementSupported`.
+`createBackgroundReplacementDevice` may throw an error if the processor was not loaded. You should check whether or not the processor has been loaded correctly by checking the state of `isBackgroundReplacementSupported`
+before calling `createBackgroundReplacementDevice`. Calling `createBackgroundReplacementDevice` will create a new processor. Users would also need to stop previously created `DeafultVideoTransformDevice` by calling `DefaultVideoTransformDevice.stop` when constructing a new
+`DefaultVideoTransformDevice` with new video processors. For more information, refer to [Video Processing APIs](https://github.com/aws/amazon-chime-sdk-js/blob/main/guides/10_Video_Processor.md#stopping-videotransformdevice);
```

Comment:
Sorry, replace `;` with `.` here too. Some markdowns were hidden in a GitHub PR tool.

Reply:
Sorry I thought I had fixed this in the last rev, should be fixed now.